book.toml

```diff
@@ -1,5 +1,6 @@
 [book]
 title = "ThreeFold Manual"
+description= "A helpful example"
```

Comment:
Could we write something else here?
Perhaps: description= "The official TFGrid documentation"